drivers/nfc/nrfxnfc/platform.c

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
You cannot include source code with the Nordic 5-clause license in Zephyr. This license is not approved by the OSI and thus it is not a candidate for inclusion.